tools/docker/fabric-all-in-one/Dockerfile_v1.4.x

```diff
@@ -41,7 +41,7 @@ RUN apk add --no-cache libc6-compat
 ENV CACTUS_CFG_PATH=/etc/hyperledger/cactus
 RUN mkdir -p $CACTUS_CFG_PATH
 # OpenSSH - need to have it so we can shell in and install/instantiate contracts
-RUN apk add --no-cache openssh augeas
+RUN apk add --no-cache augeas
```

Comment:
@outSH I think we need the OpenSSH server installed for contract deployments. Is this somehow part of the indy-validator fix or just conflict resolution gone wrong?

LGTM otherwise, but I want to make sure about this one.

Reply:
@petermetz Yes, sorry, this is leftover from temporary change from dependent commit when I was testing this change. This has been reverted, openssh remains.